addon/mixins/prop-types.js

```diff
@@ -1,8 +1,10 @@
 /**
  * The PropTypesMixin definition
  */
-import Ember from 'ember'
-const {Mixin, assign, get, getWithDefault, merge, typeOf} = Ember
+import {get, getWithDefault} from '@ember/object'
+import Mixin from '@ember/object/mixin'
+import {assign, merge} from '@ember/polyfills' // eslint-disable-line
```

Comment:
I'm assuming `// eslint-disable-line` is needed, but why so?

Reply:
That was put there by the codemod. I removed it and ran our linting and did not get an error. So it has not been removed.